Makefile.common

```diff
@@ -247,8 +248,12 @@ proto:
 
 ifdef GOLANGCI_LINT
 $(GOLANGCI_LINT):
+	-ls -lah $(FIRST_GOPATH)/bin
 	mkdir -p $(FIRST_GOPATH)/bin
-	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(FIRST_GOPATH)/bin $(GOLANGCI_LINT_VERSION)
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
```

Comment:
someone on the upstream issue commented that one could use `https://raw.../$(GOLANGCI_LINT_VERSION)/install.sh` to be even safer.

Reply:
👌